From 0a6d33ae0874e86f81ac55c7f892c9e1e0252f53 Mon Sep 17 00:00:00 2001
From: Neal Gompa <ngompa13@gmail.com>
Date: Wed, 5 May 2021 22:05:04 -0400
Subject: [PATCH] cmake: Fix to support finding PipeWire's libjack

With Linux distributions that have all audio go through PipeWire,
we want to be able to use the PipeWire version of libjack, even
though the library is installed to a non-standard path.

In particular, this impacts Fedora Linux, which is the first
distribution to have made this change.

Reference: https://fedoraproject.org/wiki/Changes/DefaultPipeWire
---
 cmake/Modules/FindJack.cmake | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/cmake/Modules/FindJack.cmake b/cmake/Modules/FindJack.cmake
index 4bdfe0a7e..bce156268 100644
--- a/cmake/Modules/FindJack.cmake
+++ b/cmake/Modules/FindJack.cmake
@@ -33,6 +33,7 @@ else (JACK_LIBRARIES AND JACK_INCLUDE_DIRS)
     NAMES
       jack/jack.h
     PATHS
+      ${_JACK_INCLUDE_DIRS}
       ${_JACK_INCLUDEDIR}
       /usr/include
       /usr/local/include
@@ -44,10 +45,15 @@ else (JACK_LIBRARIES AND JACK_INCLUDE_DIRS)
     NAMES
       jack
     PATHS
+      ${_JACK_LIBRARY_DIRS}
       ${_JACK_LIBDIR}
+      /usr/lib64
       /usr/lib
+      /usr/local/lib64
       /usr/local/lib
+      /opt/local/lib64
       /opt/local/lib
+      /sw/lib64
       /sw/lib
   )
 
-- 
2.31.1

